charts/karpenter/templates/flowschema.yaml

```diff
@@ -0,0 +1,76 @@
+{{ if .Values.flowSchema.leaderElection.create }}
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta3
+kind: FlowSchema
+metadata:
+  name: karpenter-leader-election
+  labels:
+    {{- include "karpenter.labels" . | nindent 4 }}
+  {{- with .Values.additionalAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  distinguisherMethod:
+    type: ByUser
+  matchingPrecedence: {{ .Values.flowSchema.leaderElection.matchingPrecedence }}
```

Comment:
Do you see a need from CX to customize this field?
If no, I'd suggest to use a fixed value. It can reduce the chance a user does something wrong with it.

Reply:
I think it makes sense for users to be able to customize this. Not every user may have the same precedence values so we shouldn't assume that they do, particularly if they aren't using a default EKS control plane